src/virtualenv/create/creator.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, options, interpreter):
         self._debug = None
         self.dest = Path(options.dest)
         self.clear = options.clear
+        self.vcsignore = options.vcsignore
```

Comment:
Add separator between VCS and ignore to follow the existing style.

Reply:
yes it looks Italian doesn't it

Reply:
😕